test/ci-benchmark.sh

```diff
@@ -1,9 +1,9 @@
+#!/usr/bin/env sh
```

Comment:
Ah this was a typo from me, even though I looked at this file, forgot it when copying the headers.